pkg/db/context.go

```diff
@@ -23,10 +18,8 @@ func (c *ConnectionFactory) NewContext(ctx context.Context) (context.Context, er
 		return ctx, err
 	}
 
-	// adding txid explicitly to context with a simple string key and int value
-	// due to a cyclical import cycle between pkg/db and pkg/logging
-	ctx = context.WithValue(ctx, "txid", tx.txid) //nolint
```

Comment:
This was fixed by https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1227/commits/de3c8de28fba0d5031cfc2e1cf8d701bf3df4ccd which introduced the https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/de3c8de28fba0d5031cfc2e1cf8d701bf3df4ccd/pkg/constants/db_context_keys.go to also fix the cyclic dependency issue